docs/meetings.md

```diff
@@ -0,0 +1,51 @@
+5. So-called post-standup topics are discussed. To not disturb statuses, any announcement or topic requiring a discussion/response is left to the end of the meeting.
+
+## Architecture meeting
+
+To not make standup meetings too long, we've separated the longer discussion into a weekly organised meeting. As the name suggests, the main goal of this meeting is to discuss technical or architectural topics but there is a place to discuss anything. The topics are collected beforehand in a shared document. Alternatively, anyone can add a "discuss" label to an issue.
+
+[Kanban lead](TBD) facilitates the discussion and shows a 5-minute timer for each topic (can be reset if everyone agrees to continue with a discussion). Kanban lead is also responsible for note-taking and making a clear conclusion from the discussion including the clear action items. Anyone is welcome to help with the notes since sometimes it's hard to facilitate discussion and make notes at one time.
+
+Meetings are safe space. There are no dumb ideas or stupid questions - anyone can ask if they don’t understand something.
+
+:::tip
+If there is a strong decision needed, let people provide feedback also offline (either before or after the meeting). Some people might prefer written communication and/or asynchronous communication. Allow them to participate so the best decision is chosen.
+:::
+
+## Refinement meeting
+
+Weekly meetings to prepare our next cards to work on. This is not to **_plan_** the work for the next week as done in Scrum. Kanban is based on a stream of cards going through the board columns. This activity is about moving the card from the top of the backlog (a priority one in our case) into the _refined_ column so they can be taken by anyone to start the real work.
+
+When refining, we need to make sure that the card is clearly defined (i.e. **anyone** can work on the card), there is a clear definition of done and the card is doable in 5 or fewer days.
+
+The discussion starts with a quick introduction of the card (done by a [Kanban lead](TBD) or a person the most familiar with the task). The card is being updated and the discussion continues until we agree it's clear and can be done in 5 days or less -- to help ourselves reach this agreement, there is a voting going on. We used to do a thumbs-up/down for the following questions:
+
+1. Is this card clear?
+2. Is this doable in 5 days or less?
+
+Since there wasn't much disagreement and discussion, we switched to Scrum-like voting by hand about story points. (A difference in story points helps us better find a difference in the understanding.) We collect the story points but it's not the main reason why we do the voting. Be respectful when asking people why they've chosen a different story point number. It's to help everyone with the understanding and also to share the possible risks, not to put anyone in a spot!
```

Comment:
```suggestion
Since there wasn't much disagreement and discussion, we switched to Scrum-like voting by hand about story points. (A difference in story points helps us better find a difference in the understanding.) We collect the story points but it's not the main reason why we do the voting. Be respectful when asking people why they've chosen a different story point number. It's to help everyone with the understanding and also to share the possible risks, not to put someone on the spot!
```

it's an idiom